<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ViAn</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutLeft">
        <item>
         <widget class="QTreeWidget" name="ProjectTree">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutMiddle">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <widget class="QFrame" name="videoFrameContainer">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="videoFrame">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutMiddle">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <widget class="QPushButton" name="fastBackwardButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Decrease play speed&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>&lt;&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="previousFrameButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Previous frame&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="playPauseButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Play video&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextFrameButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Next frame&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="fastForwardButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Increase play speed&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>&gt;&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Stop video&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bookmarkButton">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Bookmark&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="sliderLayout">
          <item>
           <widget class="QLabel" name="currentTimeLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="videoSlider">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="sliderPosition">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="totalTimeLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutRight">
        <item>
         <widget class="QListWidget" name="documentList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>616</width>
     <height>19</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuAdd">
     <property name="title">
      <string>Add</string>
     </property>
     <addaction name="actionAddProject"/>
     <addaction name="actionAddVideo"/>
    </widget>
    <widget class="QMenu" name="menuDelete">
     <property name="title">
      <string>Delete</string>
     </property>
     <addaction name="actionDeleteProject"/>
     <addaction name="actionDeleteVideo"/>
     <addaction name="actionDeleteAnalys"/>
     <addaction name="actionDeleteDrawing"/>
    </widget>
    <addaction name="menuAdd"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="menuDelete"/>
    <addaction name="actionExit"/>
    <addaction name="actionChoose_Workspace"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOpen_manual"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_hide_overlay"/>
    <addaction name="actionShow_hide_analysis_area"/>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="actionRotate_right"/>
    <addaction name="actionRotate_left"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuShape">
     <property name="title">
      <string>Shape</string>
     </property>
     <addaction name="actionRectangle"/>
     <addaction name="actionCircle"/>
     <addaction name="actionLine"/>
     <addaction name="actionArrow"/>
     <addaction name="actionPen"/>
     <addaction name="actionText"/>
    </widget>
    <addaction name="actionColour"/>
    <addaction name="menuShape"/>
    <addaction name="actionUndo"/>
    <addaction name="actionClear"/>
    <addaction name="actionContrast_Brightness"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar_no_overlay">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionLoad"/>
   <addaction name="actionAddVideo"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
   <addaction name="actionShow_hide_overlay"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionColour"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionCircle"/>
   <addaction name="actionArrow"/>
   <addaction name="actionLine"/>
   <addaction name="actionText"/>
   <addaction name="actionPen"/>
   <addaction name="actionUndo"/>
   <addaction name="actionClear"/>
   <addaction name="actionShow_hide_overlay"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/save_project.png</normaloff>:/save_project.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="iconText">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save project</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/load_project.png</normaloff>:/load_project.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionAddProject">
   <property name="text">
    <string>Project</string>
   </property>
   <property name="toolTip">
    <string>AddProject</string>
   </property>
  </action>
  <action name="actionAddVideo">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/add_video.png</normaloff>:/add_video.png</iconset>
   </property>
   <property name="text">
    <string>Video</string>
   </property>
   <property name="toolTip">
    <string>AddVideo</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen_manual">
   <property name="text">
    <string>Open manual</string>
   </property>
  </action>
  <action name="actionShow_hide_overlay">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/show_hide_overlay.png</normaloff>:/show_hide_overlay.png</iconset>
   </property>
   <property name="text">
    <string>Show/hide overlay</string>
   </property>
   <property name="toolTip">
    <string>Show/hide overlay</string>
   </property>
  </action>
  <action name="actionColour">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/colour.png</normaloff>:/colour.png</iconset>
   </property>
   <property name="text">
    <string>Colour</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/box.png</normaloff>:/box.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/circle.png</normaloff>:/circle.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionArrow">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/arrow.png</normaloff>:/arrow.png</iconset>
   </property>
   <property name="text">
    <string>Arrow</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/line.png</normaloff>:/line.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionDeleteProject">
   <property name="text">
    <string>Project</string>
   </property>
   <property name="toolTip">
    <string>DeleteProject</string>
   </property>
  </action>
  <action name="actionDeleteVideo">
   <property name="text">
    <string>Video</string>
   </property>
   <property name="toolTip">
    <string>DeleteVideo</string>
   </property>
  </action>
  <action name="actionDeleteAnalys">
   <property name="text">
    <string>Analys</string>
   </property>
   <property name="toolTip">
    <string>DeleteAnalys</string>
   </property>
  </action>
  <action name="actionDeleteDrawing">
   <property name="text">
    <string>Drawing</string>
   </property>
   <property name="toolTip">
    <string>DeleteDrawing</string>
   </property>
  </action>
  <action name="actionPen">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/pen.png</normaloff>:/pen.png</iconset>
   </property>
   <property name="text">
    <string>Pen</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/clear.png</normaloff>:/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/undo.png</normaloff>:/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/zoom_in.png</normaloff>:/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/zoom_out.png</normaloff>:/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="iconText">
    <string>Zoom out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionText">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/text.png</normaloff>:/text.png</iconset>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
   <property name="iconText">
    <string>Text</string>
   </property>
   <property name="toolTip">
    <string>Text</string>
   </property>
  </action>
  <action name="actionChoose_Workspace">
   <property name="text">
    <string>Choose Workspace</string>
   </property>
  </action>
  <action name="actionShow_hide_analysis_area">
   <property name="text">
    <string>Show/hide analysis area</string>
   </property>
  </action>
  <action name="actionContrast_Brightness">
   <property name="text">
    <string>Contrast/Brightness</string>
   </property>
  </action>
  <action name="actionRotate_right">
   <property name="text">
    <string>Rotate right</string>
   </property>
  </action>
  <action name="actionRotate_left">
   <property name="text">
    <string>Rotate left</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
